gridwords.py

```diff
@@ -380,8 +383,7 @@ def insertLetter(event):
     if "cell_grid" in frames_dict:
         if frames_dict["cell_grid"].mode == 'fill':
             key = event.char.upper()
-            if key in string.ascii_uppercase: #ascii_letters:
-                #print(f"DEBUG\tLetter = {event.char}")
+            if key and key >= 'A' and key <= 'Z':
```

Comment:
The problem was... `event.char.upper()` returns **empty-string** (e.g. `""`)for non-character key events, and then the code would check for existence of **empty-string** in **non-empty-string** which is always `True`.